core-primitives/types/src/lib.rs

```diff
@@ -42,16 +41,14 @@ pub type PalletString = String;
 
 pub use sp_core::{crypto::AccountId32 as AccountId, H256};
 
+pub use itp_sgx_runtime_primitives::types::*;
+
 // FIXME: Consolidate type definiton with #905.
-pub type Balance = u128;
 pub type AccountData = pallet_balances::AccountData<Balance>;
 pub type AccountInfo = frame_system::AccountInfo<Index, AccountData>;
 
 pub type ShardIdentifier = H256;
-pub type BlockNumber = u32;
 pub type Amount = u128;
```

Comment:
Eh. That's a funny type. Don't see what that should define and it doesn't really seem to be used anywhere. Can you remove this?

Reply:
It was removed from `core-primitives/types/src/lib.rs`, or did you mean that I should remove it from `sgx-runtime-primitives` as well?

Reply:
I meant the `pub type Amount = u128;`, sorry for being unclear. If it's really not used anywhere: Yes, please, remove it 😄